bitrise.yml

```diff
@@ -583,14 +583,14 @@ workflows:
         - scheme: PaymentSheet Example
         - test_plan: PaymentSheet Example-Shard2
         - log_formatter: xcbeautify
-        - xcodebuild_options: -maximum-concurrent-test-simulator-destinations 2 -maximum-parallel-testing-workers 2
+        - xcodebuild_options: -maximum-concurrent-test-simulator-destinations 4 -maximum-parallel-testing-workers 4
```

Comment:
Putting this at 6 workers causes it to fail for some reason

Reply:
We should be consistent between shards IMO. It gets flaky when we add too many, and I don't want engineers writing tests to have to micromanage the shard count

Reply:
Turned out to be an actual flake not caused by more workers, it's now fixed. Both shards are now using 6 workers. https://stripe.slack.com/archives/C02HQBW38JX/p1721675483160809